fission/util/util.go

```diff
@@ -128,10 +113,31 @@ func KubifyName(old string) string {
 	return newName
 }
 
-func GetKubernetesClient(kubeConfig string) (*restclient.Config, *kubernetes.Clientset) {
-	config, err := clientcmd.BuildConfigFromFlags("", kubeConfig)
+// GetKubernetesClient builds a new kubernetes client. If the KUBECONFIG
+// environment variable is empty or doesn't exist, ~/.kube/config is used for
+// the kube config path
+func GetKubernetesClient() (*restclient.Config, *kubernetes.Clientset) {
+	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
+
+	kubeConfigPath := os.Getenv("KUBECONFIG")
+	if len(kubeConfigPath) == 0 {
+		home := os.Getenv("HOME")
```

Comment:
To support windows better, I suggest here we can use `os/user` to get the correct home directory path.
reference: https://stackoverflow.com/a/13004756

Reply:
Thanks! I'll take a look.

Reply:
@life1347 I changed things up as per the link. Seems like a better cross-platform implementation.